README.md

```diff
@@ -24,7 +22,25 @@
 
 ---
 
-Phel is a functional programming language that compiles to PHP. It is a dialect of Lisp inspired by [Clojure](https://clojure.org/) and [Janet](https://janet-lang.org/).
+[Phel](https://phel-lang.org/) is an [FP](https://en.wikipedia.org/wiki/Functional_programming) language that compiles to PHP.
```

Comment:
You can link functional programming. But please don't write `FP` but `functional programming`. Not everybody is aware of the abbreviation.